data/policy_permissions_list/master_policy_permissions_list.yaml

```diff
@@ -212,7 +212,7 @@
       required: true
 - id: "./compliance/aws/untagged_resources/aws_untagged_resources.pt"
   name: AWS Untagged Resources
-  version: 5.1.2
+  version: 5.2.0
```

Comment:
FYI, you don't have to rerun the script that updates these yaml/json files (if it is just a version change and the policy is already in the validated list) - this should be handled automatically via a workflow that runs when this change is merged to master